args/args.go

```diff
@@ -115,7 +115,7 @@ func (args *Args) setCommandLineArgs() (err error) {
 	)
 
 	flag.StringVarP(
-		&args.SwupdMirror, "mirror", "m", args.SwupdMirror, "Swupd Installation mirror URL",
+		&args.SwupdMirror, "swupd-mirror", "m", args.SwupdMirror, "Swupd Installation mirror URL",
```

Comment:
Should we leave the short option name `-m`?
Maybe we should make it `-s` or drop it.

Reply:
had the same doubt, I'll drop it.